nesta/core/luigihacks/autobatch.py

```diff
@@ -92,7 +93,9 @@ class AutoBatchTask(luigi.Task, ABC):
     test = luigi.BoolParameter(default=True)
     max_live_jobs = luigi.IntParameter(default=25)
     worker_timeout = float('inf')
-    
+    date = luigi.DateParameter(default=dt.now())
```

Comment:
very minor addition - just means this is in here by default. there can literally be no side-effect of this, promise!